app/assets/stylesheets/views/article-form.scss

```diff
@@ -167,22 +167,23 @@
     position: sticky;
     top: 0;
     background: var(--base-0);
-    padding: 0 var(--content-padding-x);
+    padding: var(--su-2) var(--content-padding-x);
```

Comment:
This basically reverts some of the previous changes that were meant to address scrolling issue...